### DISCUSSION TITLE
10.21.0.1

### DISCUSSION THREAD

**Discussion by SubJunk** [UniversalMediaServer/UniversalMediaServer#2963] [General]
This release improves TV series metadata, improves transcoding quality and speed, and fixes so very many bugs!

Note: **Make sure you enable the cache before updating to this release, or wait for the next one.**
There is a known bug in this release that causes a startup crash if the cache is disabled. That is fixed in both `10.22.0` and `11.0.0-a2`.

	General:
		Improved API matches for TV series
		Language improvements (thanks, NCAA!)
		Fixed database not recovering from serialization changes (#2874)
		Fixed custom renderer configurations not being loaded (#2917)
		Fixed MEncoder and MPlayer on Docker (thanks, Jille Timmermans!) (#2922)
		Fixed config being overwritten when UMS closes (#2875)
		Fixed setting directory as fully played when folder names are similar
		Fixed peakaboo in status bar during scans
		Fixed database upgrade bugs
	Translation updates via Crowdin:
		German (100%) (thanks, pipin!)
		Portuguese (100%) (thanks, RPargana!)
	Dependencies:
		Bump FFmpeg to latest
		Bump h2 from 2.1.210 to 2.1.212
		Bump icu4j from 70.1 to 71.1
		Bump JRE from 8u322 to 8u332
		Bump maven-antrun-plugin from 3.0.0 to 3.1.0
		Bump maven-javadoc-plugin from 3.3.2 to 3.4.0
		Bump maven-site-plugin from 3.11.0 to 3.12.0
		Bump MediaInfo from 21.09 to 22.03
		Bump oshi-core from 6.1.5 to 6.1.6
		Bump surefire-version from 3.0.0-M5 to 3.0.0-M6

<hr /><em>This discussion was created from the release <a href='https://github.com/UniversalMediaServer/UniversalMediaServer/releases/tag/10.21.0.1'>10.21.0.1</a>.</em>

**Comment by rd1** [comment]
i'm using version 10.20.0 and when i check for updates it always says no update available Whoever's in charge of updating the version file for update checking needs to step-up their game. it's been 11 days since this release !

**Comment by schroef** [comment]
the DMG for me is unmountable?
Perhaps due to my old OSX 10.11.6
But trying it to open with other apps, it looks kinda empty

![Screen Shot 2022-05-23 at 14 31 59](https://user-images.githubusercontent.com/6923008/169884190-80e00feb-cd14-4d91-b426-95d716d51fcc.png)
![Screen Shot 2022-05-23 at 14 31 11](https://user-images.githubusercontent.com/6923008/169884195-71163525-9ccb-47b6-a312-8b035b4bbca2.png)

**Reply by SubJunk** [reply]
I saw that after leaving this comment, you created [an issue](https://github.com/UniversalMediaServer/UniversalMediaServer/issues/2994) that mentions this version, does that mean you got the DMG working?

**Comment by schroef** [comment]
That was directly downloaded from the website
